ItemDescription3/tableEXT/salesLineTabExt.50401.TabExt.al

```diff
@@ -0,0 +1,24 @@
+tableextension 50401 "BCP salesLineTabExt" extends "Sales Line"
+{
+    fields
+    {
+        modify("No.")
+        {
+            trigger OnAfterValidate()
+            begin
+                item.SetRange("No.", "No.");
```

Comment:
You forgot to check current type of lines. What if someone will create G/L Account with same number as exist Item? In this case upon g/l validation you will populate item description which is wrong.

Also instead of setrange/findfirst you should use Get. There is no point to use any find if you know exactly primary key what are you looking for.

Reply:
Hmmmm.. Okay
that'll therefore save more time it takes the get the data.